src/stores/pacbioRunCreate.js

```diff
@@ -466,7 +460,7 @@ export const usePacbioRunCreateStore = defineStore('pacbioRunCreate', {
      * Sets the defaultWellAttributes
      */
     setDefaultWellAttributes() {
-      this.defaultWellAttributes = defaultWellAttributes()
+      this.defaultWellAttributes = PacbioRunWellSmrtLinkOptions.defaultAttributes
```

Comment:
This will need to be spread `{...PacbioRunWellSmrtLinkOptions.defaultAttributes }` as we don't want to mutate the original object as we need to reset them to their original state when the run is reset.